kernel/sched.c

```diff
@@ -675,6 +676,7 @@ void _impl_k_thread_priority_set(k_tid_t tid, int prio)
 	struct k_thread *thread = (struct k_thread *)tid;
 
 	_thread_priority_set(thread, prio);
+	sys_trace_thread_priority_set(tid);
```

Comment:
Could we move this into _thread_priority_set()?

Reply:
sure